src/pages/workshops.astro

```diff
@@ -51,13 +51,29 @@ import Trainer from "../components/trainer.astro";
                 github="github.com/jamesmunns"
             />
         </div>
+        <div class="space-y-3">
+            <h4 class="text-lg">Get your ticket!</h4>
+            <p>
+                - Conference incl workshop - Individual - € 498<br />
+                - Conference incl workshop - Employer-paid - € 750<br />
+                <br />
+                Goto <a href="">Conference incl workshop tickets</a>, and choose <strong>"Postcard"</strong> when checking out.<br />
```

Comment:
I think you missed a space on "Goto"

Get this C keyword out of this Rust conference! 😄